src/main/java/org/balanpy/Main.java

```diff
@@ -7,23 +7,30 @@
 import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
 import javafx.fxml.FXMLLoader;
+import javafx.geometry.Rectangle2D;
 import javafx.scene.Node;
 import javafx.scene.Parent;
 import javafx.scene.Scene;
 import javafx.scene.control.Label;
 import javafx.scene.control.TextField;
 import javafx.scene.layout.GridPane;
+import javafx.stage.Screen;
 import javafx.stage.Stage;
 
 public class Main extends Application {
 	static ArrayList<Integer> temperaturas = new ArrayList<Integer>();
+	private static final double SCREEN_SCALING = 0.6;
+
+	private static final Rectangle2D screenBounds = Screen.getPrimary().getBounds();
+	private static final double WINDOW_WIDTH = screenBounds.getWidth() * SCREEN_SCALING;
+	private static final double WINDOW_HEIGHT = screenBounds.getHeight() * SCREEN_SCALING;
 
 	@FXML
 	private GridPane gridPane;
 
 	@Override
 	public void start(Stage primaryStage) throws Exception {
-		LoadScene(primaryStage, "/Temperaturas.fxml");
+		LoadScene(primaryStage, "/Clima.fxml");
```

Comment:
For now we load the Clima screen for testing, this will need to change as we add more working screens for the final version.